boot/freeldr/freeldr/arch/realmode/relocateFreeldr.inc

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
Suggestion: add a file license header.

Reply:
/*
 * PROJECT:     FreeLoader
 * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
 * PURPOSE:     Add some helpers functions and flags necessary for the relocation.
 * COPYRIGHT:   Copyright 2024 Daniel Victor <ilauncherdeveloper@gmail.com>
 */

Reply:
like this ^